modules/oadp-ceph-prerequisites.adoc

```diff
@@ -13,4 +13,4 @@ The following prerequisites apply to all back up and restore operations of data
 * You have installed the OADP Operator.
 * You have created a secret `cloud-credentials` in the namespace `openshift-adp.`
 * You have installed {rh-storage-first}.
-* You have installed the latest VolSync Operator using the Operator Lifecycle Manager.
+* You have installed the latest VolSync Operator using Operator Lifecycle Manager.
```

Comment:
I know this change isn't related to your PR but would you mind fixing this to adhere to ISG?
```suggestion
* You have installed the latest VolSync Operator by using Operator Lifecycle Manager.
```